cmd/gen/makefile/runner.go

```diff
@@ -45,3 +52,19 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 
 	return nil
 }
+
+func mapFlavourTypeToMakeFileFlavour(f string) int {
```

Comment:
Map the command flags values to `makefile` package-specific constants, to allow a looser connection between the command and the package